routes/eth.go

```diff
@@ -422,3 +514,10 @@ func (fes *APIServer) GetETHTransactionByHash(hash string) (_tx *InfuraTx, _err
 	}
 	return response, nil
 }
+
+func TextAndHash(data []byte) ([]byte, string) {
+	msg := fmt.Sprintf("\x19Ethereum Signed Message:\n%d%s", len(data), string(data))
+	hasher := sha3.NewLegacyKeccak256()
+	hasher.Write([]byte(msg))
+	return hasher.Sum(nil), msg
+}
```

Comment:
Maybe we use the lib.TextAndHash function here?

Reply:
Function isn't used so we can just remove it